docs/UserGuide.md

```diff
@@ -758,11 +797,63 @@ Exit the application.
+> `export {file_path} [/overwrite]`
+
+Export all modules progress to a JSON-format file. 
+
+- <span style="color:#e46c0a">`file_path`</span> : The path of the file to export the modules progress to
+  - Must be relative to Le Tracker's default saving directory (:exclamation:The default saving directory is `[JAR 
+    file location]/data`)
+  - Must not coincide with Le Tracker's current tracker file path. (:exclamation:The default tracker file path is `
+    [JAR file location]/data/letracker.json`)
+  - If `/overwrite` is not specified, the file specified in `file_path` must not exist
+- <span style="color:#e46c0a">`/overwrite`</span> : If specified, Le Tracker will overwrite all data in 
+  the file specified in `file_path` with the current modules progress in Le Tracker
+  - If the file specified in `file_path` exists, it must allow reading and writing operations
+  - If the file specified in `file_path` exists, it must be of JSON format
+  - If the file specified in `file_path` exists, the flag `/overwrite` will be ignored
+
+Examples: 
+
+- `export hello.json`
+
+- `export /../../haha.json /overwrite`
 
 ### Import Data
 
-<!-- TODO: fill this in -->
+> `import {file_path} [/mod {module_1}[, {module_2}[, {module_3}, ...]]] [/overwrite]`
+
+Import modules progress from a JSON-format file to the current tracker.
+
+- <span style="color:#e46c0a">`file_path`</span> : The path of the file to export the modules progress from
+  - Must be relative to Le Tracker's default saving directory (:exclamation:The default saving directory is `[JAR
+    file location]/data`)
+  - The file specified in `file_path` must exist. (:exclamation: If only the file's name is specified, the file must 
+    exist in the default saving directory at `[JAR file location]/data`)
+  - Must allow reading and writing operations
+  - Must be of JSON format
+  - Must be a valid Le Tracker data file
+- <span style="color:#e46c0a">`module_1, module_2, ...`</span> : If specified, Le Tracker will only import progress 
+  of these modules from the file specified in `file_path`
+  - If unspecified, Le Tracker will import progress of all modules in the file specified in `file_path`
+  - If `overwrite` is not specified, `module_1, module_2, ...` must not exist in the current tracker 
+  - Must belong to existing modules in the file specified in `file_path` (:exclamation: Module matching is case 
+    sensitive)
+  - Repeated modules (if any) will be ignored
+- <span style="color:#e46c0a">`/overwrite`</span> : If specified, Le Tracker will overwrite existing modules 
+  progress with the progress of the imported modules, provided they have the same code (:exclamation: Module matching is case sensitive)
+  - If the imported modules do not exist in the current tracker, the flag `overwrite` will be ignored
+
+Examples:
+
+- `import hello.json`
+
+- `import /../../haha.json /overwrite`
+
+- `import hehe.json /mod CS2040, MA2401`
+
+- `import hihi.json /mod EG2310 /overwrite`
```

Comment:
Just to check, your intention of adding empty lines between examples is because you want more spacing between them?

Reply:
I added the empty lines so it's easier to read when modifying the UserGuide.md

Reply:
Oh, on IntelliJ, it doesn't render the space, but here it does. I have deleted the space